core/src/models/RecordProperty.ts

```diff
@@ -25,7 +30,10 @@ const STATE_TRANSITIONS = [
 ];
 
 @Table({ tableName: "recordProperties", paranoid: false })
-export class RecordProperty extends LoggedModel<RecordProperty> {
```

Comment:
Not 100% sure we'll be happy with the decision if `RecordProperty` should share any behaviors now or in the future with all the other ones. Maybe `LoggedModel` should really be called `CommonModel` and logging is turned off. Something to keep an eye on.

Reply:
For example, id generation, timestamps, etc

Reply:
Yeah - we've got ~6 models now that don't extend Logged Model.  I like that though - I'll make a chain `CommonModel -> LoggedModel -> ActualModel`